src/search/pruning_method.cc

```diff
@@ -13,6 +16,31 @@ PruningMethod::PruningMethod()
 void PruningMethod::initialize(const shared_ptr<AbstractTask> &task_) {
     assert(!task);
     task = task_;
+    num_unpruned_successors_generated = 0;
+    num_pruned_successors_generated = 0;
+}
+
+void PruningMethod::prune_op_ids(
+    const State &state, vector<OperatorID> &op_ids) {
+    assert(!task_properties::is_goal_state(TaskProxy(*task), state));
+    timer.resume();
+    int num_ops_before_pruning = op_ids.size();
+    prune_operators(state, op_ids);
+    num_unpruned_successors_generated += num_ops_before_pruning;
+    num_pruned_successors_generated += op_ids.size();
+    timer.stop();
+}
+
+void PruningMethod::print_statistics() const {
+    utils::g_log << "total successors before partial-order reduction: "
```

Comment:
Are all pruning methods partial-order reductions? Even if this is (currently) the case, what speaks against simply saying "total successors before/after pruning"?